### ISSUE TITLE
Return generic UI if options.public is not set

### ISSUE THREAD

**Issue by gr2m** [eHealthAfrica/kazana#195]
for a better first-time run experience with Kazana, it would be cool if we could load a generic default app at the root path if `options.public` is not set in the kazana manifest.

**Comment by karl-ehealth** [comment]
one size doesn't fit all here. GCUA is not using public, that handler got in the way of our server-rendering handler, so we had to specify our own routes for static files https://github.com/eHealthAfrica/guinea-connect-universal-app/blob/master/src/kazana/routes.js#L2

**Comment by gr2m** [comment]
it would be just a default if `options.public` is not set. Better than showing nothing

**Comment by karl-ehealth** [comment]
Yeah like i said, we're not setting `options.public`. When you add an options.public, you get a route with a catch-all splat handler that looks like this `/{path*}`. However we need exactly that splat handler for our server-side rendering (see https://github.com/eHealthAfrica/guinea-connect-universal-app/blob/master/src/kazana/routes.js#L33), so if something would default to try to register that route (like i said we don't use options.public cause we can't), the server would throw an error when we register our SSR route.

**Comment by gr2m** [comment]
ah gotcha, sorry missed your point, that makes totally sense. One idea we had would be to allow `options.public` to not only be set to a path string, but also to a hapi plugin which would define routes like in https://github.com/eHealthAfrica/guinea-connect-universal-app/blob/master/src/kazana/routes.js 

That way we could build re-usable kazana apps that could be easily used instead of the kazana default UI, until a custom app is built (if needed at all)

**Comment by karl-ehealth** [comment]
interesting idea!

**Comment by jo** [comment]
Link to example html mentioned in README: https://gist.github.com/gr2m/58c3bd81412779b81b91

**Comment by gr2m** [comment]
Note that [this gist](https://gist.github.com/gr2m/58c3bd81412779b81b91) is just a copy & paste from https://github.com/eHealthAfrica/kazana-example
